table.go

```diff
@@ -490,8 +490,8 @@ func (t *TableBlock) splitGranule(granule *Granule) {
 		return
 	}
 
-	// Obtain a new tx for this compaction
-	tx, watermark, commit := t.table.db.begin()
+	// Use the latest watermark as the tx id
```

Comment:
This is safe to do because we're not actually adding new data, we're just compacting it, right?